.github/workflows/nightly_tests.yaml

```diff
@@ -95,6 +95,12 @@ jobs:
       run : python3 xpk.py info --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
     - name: Delete the workload on the cluster
       run: python3 xpk.py workload delete --workload $WORKLOAD_NAME --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
+    - name: Run a batch job on the cluster
+      run: python3 xpk.py batch --cluster $TPU_CLUSTER_NAME --zone=us-central2-b test.sh
+    - name: Get job info for the last job created on the cluster
+      run: |
+        export MOST_RECENT_JOB_NAME=$(kubectl get jobs --sort-by=.metadata.creationTimestamp -o jsonpath="{.items[-1:].metadata.name}") 
+        python3 xpk.py job info $MOST_RECENT_JOB_NAME | grep "Exiting XPK cleanly"
```

Comment:
grep EXiting xpk cleanly makes no sensie. If xpk does not exits cleany it will exit with code != 0 and pipeline will fail

Reply:
Right, forgot that the command itself exists with a number.